test/browser/browser-test.js

```diff
@@ -119,17 +119,25 @@ module.exports = function(localIdentifier, PORT, capabilityMatrix) {
           // query it in parallel:
           // https://github.com/SeleniumHQ/selenium/issues/422#issuecomment-90629726
           const titles = [];
+          console.log('DEBUG START ITERATING over n elements', elements.length); // eslint-disable-line
```

Comment:
Draft PR, I know, but this caught my eye.  What eslint error is this suppressing?  (we have the `no-console` turned off, right?)

FWIW, I believe you can suppress a rule for a whole file by adding a comment like `/* eslint no-console: 0 */` at the top of the file.

Reply:
Sorry, this just comes from one of my vim snippets that insert `console.log(); // eslint-disable-line`. I mainly use that in other projects where `no-console` is not turned off.